main.py

```diff
@@ -31,23 +33,32 @@
     'o': '#fabd2f'
     }
 
+colors_list = ['b', 'B', 'g', 'G', 'p', 'P', 'r', 'R', 'y', 'Y', 'o', 'O']
+
+def turtle_starting_pos(i):

```

Comment:
The fucntion is not pure. If the indexes change, the function would target a wrong turtle. Maybe it should accept a turtle object as a param?
[Pure functions](https://realpython.com/python-functional-programming/#what-is-functional-programming)